README.md

```diff
@@ -68,13 +72,13 @@ storyblokInit({
   components: {
     page: Page,
     teaser: Teaser,
-    grid: Grid,
-    feature: Feature,
   },
 });
 ```
 
-> Add all your components to the components object in the `storyblokInit` function.
+> Note: This is the general way for initalizing the SDK, the initialization might be different depending upon the framework. You can see how everything works according to the framework in their respective sections below.
```

Comment:
As we're only showing the `storyblokInit` call here, I think it will be pretty much the same for all the frameworks. What it would change is where/when to call it, right?

Reply:
In the last case, where we enable the live editing we don't add the components and initialize it twice.

Reply:
👍